src/diffusers/models/unet_2d_condition.py

```diff
@@ -388,6 +437,7 @@ def forward(
         if attention_mask is not None:
             attention_mask = (1 - attention_mask.to(sample.dtype)) * -10000.0
             attention_mask = attention_mask.unsqueeze(1)
+            cross_attention_kwargs = {'prepare_mask':False}
```

Comment:
Why do we need this?

Reply:
this is to pass a flag to `CrossAttnProcessor` to skip the `prepare_attention_mask` step(I think this step is intended to use for mask in unclip model only ) 

https://github.com/huggingface/diffusers/blob/main/src/diffusers/models/cross_attention.py#L248

Reply:
Actually the model doesn't use an attention mask I think or it simply doesn't make any difference